README.md

```diff
@@ -39,69 +43,131 @@ Pre-built docker for popular AI frameworks. Easy to include heterogeneous hardwa
+OpenPAI is a most complete solution for deep learning, support virtual cluster, compatible Hadoop / Kubernetes eco-system, complete training pipeline at one cluster etc. OpenPAI is architected in a modular way: different module can be plugged in as appropriate.
 
 ## Related Projects
+
 Targeting at openness and advancing state-of-art technology, [Microsoft Research (MSR)](https://www.microsoft.com/en-us/research/group/systems-research-group-asia/) had also released few other open source projects.
+
 * [NNI](https://github.com/Microsoft/nni) : An open source AutoML toolkit for neural architecture search and hyper-parameter tuning.
 We encourage researchers and students leverage these projects to accelerate the AI development and research.
 * [MMdnn](https://github.com/Microsoft/MMdnn) : A comprehensive, cross-framework solution to convert, visualize and diagnose deep neural network models. The "MM" in MMdnn stands for model management and "dnn" is an acronym for deep neural network.
 
-## How to deploy
-#### 1 Prerequisites <a name="ref_prerequisites"></a>
-Before start, you need to meet the following requirements:
+## Try OpenPAI
+
+To try OpenPAI quickly, you can follow [Installation](#Installation) below, and skip customization part. You can add more servers as needed after deployed if you keep OpenPAI for using.
+
+It's highly recommended to try OpenPAI on server(s), which has no usage and service. You can deploy OpenPAI on one or multiple server(s), please refer to [here](https://github.com/Microsoft/pai/wiki/Resource-Requirement) for hardware specification.
+
+## Installation
+
+Follow this part to check prerequisites, deploy and validate an OpenPAI cluster.
+
+### Prerequisites
+
+* Ubuntu 16.04 (18.04 should work, but not fully tested.)
+* Assign each server a static IP address, and servers can communicate each other.
+* Server can access internet, especially need to have access to the docker hub registry service or its mirror. Deployment process will pull Docker images of OpenPAI.
+* SSH service is enabled and share the same username/password and have sudo privilege.
+* NTP service is enabled.
+* Recommend not to install docker or docker's version must be higher than 1.26.
+* OpenPAI reserves memory and CPU for service running, so make sure there are enough resource to run machine learning jobs. Check [hardware requirements](https://github.com/Microsoft/pai/wiki/Resource-Requirement) for details.
+* Dedicated servers for OpenPAI. OpenPAI manages all CPU, memory and GPU resources of servers. If there is any other workload, it may cause unknown problem due to insufficient resource.
+
+### Deploy
+
+The default deployment includes minimum steps to deploy an OpenPAI cluster. Base on the default configuration, you can customize OpenPAI deployment for different hardware environments and use scenarios.
+
```

Comment:
As discussed, please add the previous "Single box deployment" here. maybe we can rename it to "Reference minimal installation".

Reply:
added